src/components/form-authentication-components/Login.tsx

```diff
@@ -1,19 +1,40 @@
-import { useFormik } from 'formik';
 import React, { useEffect, useState } from 'react';
+import { useFormik } from 'formik';
+import axios from 'axios';
 import { initialValues, loginSchema } from './loginData';
 import { Input } from './Input';
+import { useNavigate } from 'react-router-dom';
+import { ToastTheme, Logintype, setLoginType } from './types';
 
-type setLoginType = {
-	setLogin: React.Dispatch<React.SetStateAction<string>>;
-};
+export const Login = ({
+	setLogin,
+	setNotifyMessage,
+	setNotifyTheme,
+}: setLoginType) => {
+	const navigate = useNavigate();
 
-type Logintype = {
-	email: string;
-	password: string;
-};
+	const handleLogin = async () => {
+		axios
+			.post('http://localhost:4000/login', loginData)
+			.then((response) => {
+				const token = response.data.token;
+				const email = response.data.email;
+				sessionStorage.setItem('token', token);
+				sessionStorage.setItem('email', email);
+
+				setbtnLoginSuccess(true);
+				setNotifyMessage(response.data.message);
+				setNotifyTheme(ToastTheme.SUCCESS);
 
-export const Login = ({ setLogin }: setLoginType) => {
-	const handleLogin = async () => {};
+				setTimeout(() => {
+					navigate('/invoices');
+				}, 5000);
```

Comment:
po loginie to ten timeout jest niepotrzebny, przekieruj uzytkownika od razu :)

Reply:
zmniejsze ten timeout do 1,5sek - chodzi o to zeby toastify czyli to powiadomienie zniknelo